ci_example_python/doc/python_code_guidelines.md

```diff
@@ -0,0 +1,46 @@
+Coding Guidelines {#coding_guidelines_0}
+=================
+
+## I. Introduction
+
+These are the internal **Python** guidelines for the
+[machines-in-motion](https://wp.nyu.edu/machinesinmotion/) group. The same
+guidelines are used in the
+[Open Dynamic Robot Initiative](https://open-dynamic-robot-initiative.github.io/)
+
+The following rules present basic guidelines for our **Python** code.
+The goal is to have code that is formatted in a consistent and easily readable
+way while at the same time not being overly complicated by specifying every
+detail. For such guidelines to be practical, newcomers should be able to read
+them within a few minutes and be able to memorize them. So these rules
+intentionally do not cover every detail but rather aim at specifying only the
+big, important things.
+
+If this is too simple for you and you want more rules, you are encouraged to
+read the [PEP 8](https://www.python.org/dev/peps/pep-0008/) on which these rules
+are based.
+
+These guidelines may evolve in time so it is first
+good practice to check them upon creation of a new package or code refactoring.
+
+## II. Folder Structure and File Naming
+
+- Only one **Python** module per git repository, and it must be located in
+    `python/<package_name>/`.
```

Comment:
Maybe we should add that the package shall actually have the same name as the catkin_package it belongs to.

Reply:
I can change `<package_name>` by `<catkin_package_name>`

Reply:
yeah, that should be clear :+1: 

Another thing I just noticed: I think it should be "Only one Python _package_". If I got it correctly the directory is a package while each file in it is a module.